src/con_duct/__main__.py

```diff
@@ -126,14 +127,29 @@ class ProcessStats:
     rss: int  # Memory Resident Set Size in Bytes
     vsz: int  # Virtual Memory size in Bytes
     timestamp: str
+    etime: str
+    cmd: str
 
     def max(self, other: ProcessStats) -> ProcessStats:
+        cmd = self.cmd
+        if self.cmd != other.cmd:
+            lgr.debug(
+                f"cmd has changed. Previous measurement was {self.cmd}, now {other.cmd}."
+            )
+            # Brackets indicate that the kernel has substituted an abbreviation.
+            surrounded_by_brackets = r"\[.*?\]"
```

Comment:
why to overcomplicate (the `?`) and shouldn't it match at the beginning? otherwise it could be some argument etc.  And should be at least one character I think

```suggestion
            surrounded_by_brackets = r"^\[.+\]"
```

Reply:
+1